doc/api/http.md

```diff
@@ -917,6 +917,32 @@ Example:
 var contentType = response.getHeader('content-type');
 ```
 
+### response.getHeaders([namesOnly])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `namesOnly` {Boolean}
+* Returns: {Object | Array}
+
+Returns a (shallow) copy of the current outgoing headers. If `namesOnly` is
+`true`, then only an array of the header names are returned. Otherwise a plain
+object is returned whose keys are the header names and whose values are the
+respective header values. All header names in the returned value are lowercase.
+
+Example:
+
+```js
+response.setHeader('Foo', 'bar');
+response.setHeader('Set-Cookie', ['foo=bar', 'bar=baz']);
+
+var headerNames = response.getHeaders(true);
```

Comment:
Probably, although the rest of the document is still a mixture of `const` and `var`.